content/docs/command-reference/get-url.md

```diff
@@ -34,23 +33,33 @@ directory will be placed inside.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Description    | `url` format                               |
-| ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/data`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
-| `gs`    | Google Storage | `gs://mybucket/data`                       |
-| `ssh`   | SSH server     | `ssh://user@example.com/path/to/data`      |
-| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+| Type     | Description                  | `url` format example                                       |
+| -------- | ---------------------------- | ---------------------------------------------------------- |
+| `s3`     | Amazon S3                    | `s3://bucket/data`                                         |
+| `azure`  | Microsoft Azure Blob Storage | `azure://container/data`                                   |
+| `gdrive` | Google Drive                 | `gdrive://<folder-id>/data`                                |
+| `gs`     | Google Cloud Storage         | `gs://bucket/data`                                         |
+| `ssh`    | SSH server                   | `ssh://user@example.com/path/to/data`                      |
+| `hdfs`   | HDFS to file\*               | `hdfs://user@example.com/path/to/data.csv`                 |
+| `http`   | HTTP to file\*               | `https://example.com/path/to/data.csv`                     |
+| `webdav` | WebDav to file\*             | `webdavs://example.com/public.php/webdav/path/to/data.csv` |
```

Comment:
is `public.php` is a very common thingy for webdav?

Reply:
I'm not familiar TBH but apparently yes there's some sort of endpoint. Not sure if it's typically PHP. I took this from https://dvc.org/doc/command-reference/remote/add#supported-storage-types

But let me check, maybe it's quick to figure this out. ⌛

Reply:
From https://linuxconfig.org/webdav-server-setup-on-ubuntu-linux and https://docs.microsoft.com/en-us/iis/install/installing-publishing-technologies/installing-and-configuring-webdav-on-iis WebDAV is setup similar to any HTTP server so no special use of PHP here. We should probably review all these in a separate PR though, so merging this for now (will create a ticket).

Reply:
`public.php` is a public endpoint for Nextcloud and Owncloud. There's `remote.php` as a private endpoint. But, it can be removed from most of the places, but good to keep one as an example, as most of our WebDAV users are either using Owncloud or Nextcloud.